workflows/kf_single_cell_10x.cwl

```diff
@@ -110,10 +118,16 @@ steps:
       cpus: cpus
     out: [score_histogram, doublets_file]
 
-  merge:
+  seurat_merge:
```

Comment:
Let's make a ticket to see if the seurat merge step is needed since this is only running on a single sample. The script might be doing other stuff that we want to run anyway.

Reply:
Good idea!